### PR TITLE
Clear WebView's cache on Android to avoid interference with authentication methods

### PR DESCRIPTION
**Pull request by nirvn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for eb3f775e7b0a5ab945b3b077e02e6e018459b614: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-arm64-android.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-arm-neon-android.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-x64-android.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-x86-android.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-arm64-android.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-arm-neon-android.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-x64-android.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-4610-634139550d06ac2f9711d0828ac4c6f380bf9fe8-x86-android.apk)
</details>

File: src/qml/BrowserPanel.qml [hunk 1]
```diff
@@ -61,14 +64,58 @@ Popup {
...
       browserView.anchors.fill = undefined
       browserView.url = url
       browserView.opacity = 0
     }
+
+    clearCookiesOnOpen = false
+  }
+
+  function deleteCookies() {
+    console.log(browserCookies)
```
Comment (suricactus) [inline src/qml/BrowserPanel.qml:115]: Do we need this `console.log`?

File: src/qml/BrowserPanel.qml [hunk 2]
```diff
@@ -61,14 +64,58 @@ Popup {
...
+    for(var i = 0; i < browserCookies.length; i++) {
+      browserView.deleteCookie(browserCookies[i][0], browserCookies[i][1])
+    }
```
Comment (suricactus) [inline src/qml/BrowserPanel.qml:118]:
```suggestion
    for(const [cookie, url] of browserCookies) {
      browserView.deleteCookie(browserCookies[i][0], browserCookies[i][1])
    }
```
Isn't it more readable like this?

**Review by suricactus — APPROVED** [review]
Code looks good. There is a forgotten console.log and a tiny js improvement (not sure if that JS is supported in QML though).

**Comment by m-kuhn** [comment]
> Code looks good. There is a forgotten console.log and a tiny js improvement (not sure if that JS is supported in QML though).

Qml engine is v8 iirc

**Pull request merged**